tests/selenium/test_search_scenarios.py

```diff
@@ -1,14 +1,15 @@
 import re
 
 import pytest
+from data_platform_catalogue.search_types import ResultType
+
+from tests.conftest import generate_page, mock_search_response
 
 from .helpers import check_for_accessibility_issues
-from tests.conftest import mock_search_response, generate_page
-from data_platform_catalogue.search_types import ResultType
 
 
 @pytest.mark.slow
-class TestSearchWithoutJavascriptAndCss:
+class TestSearch:
```

Comment:
Turns out this does actually include assets. The `live_server` fixture decides whether to include assets based on the presence of `django.contrib.staticfiles` in `INSTALLED_APPS`.